src/dstack/_internal/core/services/configs/__init__.py

```diff
@@ -127,7 +126,9 @@ def update_default_project(
             (
                 default_project is None
                 or default
-                or confirm_ask(f"Update the default project in [code]{config_dir}[/]?")
+                or confirm_ask(
+                    f"Update the [code]{project_name}[/] project in [code]{config_dir}[/]?"
```

Comment:
(nit) I thought here we were asking whether to set the project as default. If we are actually asking whether to update it at all, then we might want to rename the `set_it_as_default` variable. Or remove it and put all the conditions in the `if` above